cli/flags.rs

```diff
@@ -127,10 +128,19 @@ To get help on the another subcommands (run in this case):
 
   deno help run")
     .arg(
+      Arg::with_name("log-level")
+        .short("L")
+        .long("log-level")
+        .help("Set log level")
+        .takes_value(true)
+        .possible_values(&["debug", "info"])
+        .global(true),
+    ).arg(
       Arg::with_name("log-debug")
```

Comment:
Can we just remove this one now?

Reply:
We can, but then for debug you have to go with `-L=debug`, are you fine with that?

Reply:
Yes, I prefer that to having an extra flag